horace_core/sqw/@sqw/tobyfit_DGdisk_resconv.m

```diff
@@ -115,16 +117,63 @@
 refine_crystal = ~isempty(xtal);
 refine_moderator = ~isempty(modshape);
 if refine_crystal && refine_moderator
-    error('Cannot refine both crystal and moderator parameters. Error in logic flow - this should have been caught')
+    error('HORACE:tobyfit_resconv:invalid_argument', ...
+          'Cannot refine both crystal and moderator parameters. Error in logic flow - this should have been caught')
 end
 
-
 % Initialise output arguments
 % ---------------------------
 wout = copy(win);
 state_out = cell(size(win));    % create output argument
 store_out = [];
 
+% Perform resolution broadening calculation
+% -----------------------------------------
+% Package parameters as a cell for convenience
+if ~iscell(pars)
+    pars={pars};
+end
+
+reset_state=caller.reset_state;
+
+% Factor 10 because of 44 x npix array
```

Comment:
not sure "because" is correct. It's a provisional one-size fits all value. You may wish to refer to the pr number or paste your comment from the pr here